en_US/data-integration/data-bridge-kafka-consumer.md

```diff
@@ -20,7 +20,6 @@ The following steps guide you to create data bridge to Kafka in consumer mode.
 
 - Knowledge about EMQX data integration [rules](./rules.md)
 - Knowledge about [data bridge](./data-bridges.md)
-- Relevant Kafka topics should be created before creating the data bridge.
```

Comment:
I think this is still relevant for this bridge.

Reply:
It is mentioned in the "Create Kafka Topics" section, before "Create Kafka Data Bridge".